gitlabform/gitlabform/core.py

```diff
@@ -280,6 +291,11 @@ def process_all(self, projects_and_groups, groups):
                 logging.error("+++ Error while processing '%s'", group)
                 traceback.print_exc()
 
+            if self.terminate_after_error:
```

Comment:
Ok I thought because it is a flag that has to be thrown we will exit out of the application. If the flag isn't thrown then it will continue with an error. I didn't know I was allowed to change it to where it exits out all together if there's an error regardless if the flag is thrown. I'm glad I am getting close. Thank you for being patient with me as I am trying to grow and get better.